README.md

```
@@ -1,2 +1,430 @@
+    access-key: your-s3-access-key
+    secret-key: your-s3-secret-key
+    bucket: your-s3-bucket-name
+    region: ap-northeast-2
+
+openai:
+  api-key: your-openai-api-key
+```
+
+## ▶️ 앱 실행
+
+**개발 모드 실행**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)